app/src/main/java/org/hwyl/sexytopo/control/graph/GraphView.java

```diff
@@ -1003,13 +997,11 @@ private void drawLegs(Canvas canvas, Space<Coord2D> space, int baseAlpha) {
                 paint = fade ? fadedLegPaint : legPaint;
             }
 
-            if (projectionType.isLegInPlane(leg)) {
-                canvas.drawLine((float)start.x, (float)start.y, (float)end.x, (float)end.y, paint);
+            if (leg.hasDestination() && !projectionType.isLegInPlane(leg)) {
```

Comment:
Drawing splays that are out of the plane as dashed lines is supposed to happen (I think). Lets you see which measurements are shots up an aven or part of x-sections. You're usually drawing the walls in according to what's on your plane.

Reply:
Something I've thought about but never bothered to play with to see if it feels right: have dotted lines for lines away from you (downwards on plan) and dashed lines coming towards you (upwards on plan).